simulariumio/tests/converters/test_trajectory_converter.py

```diff
@@ -59,1011 +62,1180 @@ def mixed_agents_wrong_display_type2():
 
 
 def mixed_agents_invalid_agent_id():
+    # Agent IDs are larger than a 32 bit int can represent
     result = mixed_agents()
     result.agent_data.unique_ids[0][0] = MAX_AGENT_ID + 1
     return result
 
 
+# 3 default agents (radius 5-10) at given positions for 3 frames,
+# test string for plots
+default_agents_trajectory = three_default_agents()
+default_agents_converter = TrajectoryConverter(default_agents_trajectory)
+default_agents_data = JsonWriter.format_trajectory_data(default_agents_converter._data)
+
+# 2 default agents (radius 5-10) and 3 fiber agents
+# at given positions for 3 frames, no plots
+mixed_agents_trajectory = mixed_agents()
+mixed_agents_converter = TrajectoryConverter(mixed_agents_trajectory)
+mixed_agents_data = JsonWriter.format_trajectory_data(mixed_agents_converter._data)
+
+# 3 fiber agents with points drawn
+# at given positions for 3 frames, no plots
+fiber_agents_trajectory = fiber_agents()
+fiber_agents_converter = TrajectoryConverter(fiber_agents_trajectory)
+fiber_agents_data = JsonWriter.format_trajectory_data(fiber_agents_converter._data)
+
+
+# test trajectory info
```

Comment:
After reading through the "test trajectory info" section I think you should make two new sets of data: one that has nothing set in trajectory info that is not required. And one that has everything in the trajectory info set. There is no reason for these to sets of agent data to have anything different about the agents because you're not testing that anyway in these unit tests. They could have the same 1 agent in them with no display data. That way you minimize any change that doesn't have to do with trajectory data breaking these tests.